chapters/th/_toctree.yml

```diff
@@ -0,0 +1,173 @@
+- title: 0. Setup
+  sections:
+  - local: chapter0/1
+    title: Introduction
+
+- title: 1. Transformer models
+  sections:
+  - local: chapter1/1
+    title: Introduction
+  - local: chapter1/2
+    title: Natural Language Processing
+  - local: chapter1/3
+    title: Transformers, what can they do?
+  - local: chapter1/4
+    title: How do Transformers work?
+  - local: chapter1/5
+    title: Encoder models
+  - local: chapter1/6
+    title: Decoder models
+  - local: chapter1/7
+    title: Sequence-to-sequence models
+  - local: chapter1/8
+    title: Bias and limitations
+  - local: chapter1/9
+    title: Summary
+  - local: chapter1/10
+    title: End-of-chapter quiz
+    quiz: 1
+
+- title: 2. Using 🤗 Transformers
+  sections:
+  - local: chapter2/1
+    title: Introduction
+  - local: chapter2/2
+    title: Behind the pipeline
+  - local: chapter2/3
+    title: Models
+  - local: chapter2/4
+    title: Tokenizers
+  - local: chapter2/5
+    title: Handling multiple sequences
+  - local: chapter2/6
+    title: Putting it all together
+  - local: chapter2/7
+    title: Basic usage completed!
+  - local: chapter2/8
+    title: End-of-chapter quiz
+    quiz: 2
+
+- title: 3. การ fine-tune โมเดลที่ผ่านการเทรนมาแล้ว (pretrained model)
+  sections:
+  - local: chapter3/1
+    title: บทนำ
+  - local: chapter3/2
```

Comment:
Since we don't yet have the translated files for `chapter3/2` etc, we should also remove these entries here

Reply:
Got it! Fixed accordingly